consensus/src/chained_bft/block_storage/block_store.rs

```diff
@@ -184,6 +184,7 @@ impl<T: Payload> BlockStore<T> {
                 )
             })
             .collect();
+        // TODO: if the await returns a failure, propagate it. panic is not an option.
```

Comment:
@zekun000 what's the alternative here? It sounds like if we can't commit a block that must be committed, we can't make progress either. I'm not sure what else can be done?

Reply:
There is always something else one can do than panic. If, in fact, this panic will never ever happen then change this to assume_unreachable.